sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_document_model_administration_client.py

```diff
@@ -648,7 +649,7 @@ def begin_build_document_classifier(
         """Build a document classifier. For more information on how to build and train
         a custom classifier model, see https://aka.ms/azsdk/formrecognizer/buildclassifiermodel.
 
-        :param doc_types: Required. Mapping of document types to classify against.
+        :param doc_types: Mapping of document types to classify against.
         :paramtype doc_types: Mapping[str, ~azure.ai.formrecognizer.ClassifierDocumentTypeDetails]
```

Comment:
The type for this param isnt rendering in the docs :/

Reply:
how so?

Reply:
I think the parameter here should have type info right?
<img width="676" alt="image" src="https://github.com/Azure/azure-sdk-for-python/assets/9859037/8b5014ad-9f39-410b-9a01-ff0bae157e89">

Reply:
Oh I see. Since it's a param, we need to use `:type:`

```suggestion
        :type doc_types: Mapping[str, ~azure.ai.formrecognizer.ClassifierDocumentTypeDetails]
```

Reply:
Ahh got it, updated!